t/proxyunits.t

```diff
@@ -680,15 +680,164 @@ check_sanity($ps);
 }
 
 check_sanity($ps);
+
 # Test Lua response API
-#{
-    # elapsed()
-    # ok()
-    # hit()
-    # vlen()
-    # code()
-    # line()
-#}
+{
+    subtest 'response:elapsed() >100000micros' => sub {
+        # ps_recv must not receive an error
+        my $be = $mbe[0];
+        my $cmd = "mg /response/elapsed\r\n";
+        print $ps $cmd;
+        is(scalar <$be>, $cmd, "be received request.");
+        sleep 0.1;
+        print $be "HD\r\n";
+        is(scalar <$ps>, "HD\r\n", "proxy received HD");
+    };
+
+
+    subtest 'response:ok()' => sub {
+        # ps_recv must not receive an error
+        proxy_test(
+            ps_send => "mg /response/ok\r\n",
+            be_recv => {0 => ["mg /response/ok\r\n"]},
+            be_send => {0 => ["HD\r\n"]},
+            ps_recv => ["HD\r\n"],
+        );
+    };
+
+
+    subtest 'response:hit()' => sub {
+        # ps_recv must not receive an error
+        proxy_test(
+            ps_send => "mg /response/ok\r\n",
```

Comment:
double testing ok() instead of hit here.

should also test cases where ok/hit are false work properly.